generators/client/templates/react/src/main/webapp/app/typings.d.ts.ejs

```diff
@@ -16,6 +16,15 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -%>
+
+// TODO: Fix in react-jhipster
+// Reason: getRoles from './user-management.reducer';
```

Comment:
may be we should use a custom type for getRoles?

Reply:
yes, that's probably better than making id optional for every ICrudGetAction

Reply:
it seems `getRoles` doen't do much , it fetches `${apiUrl}/authorities` but nothing is done with the result